java-client/src/main/java/org/opensearch/client/opensearch/_types/analysis/LanguageAnalyzer.java

```diff
@@ -83,7 +84,7 @@ public static LanguageAnalyzer of(Function<Builder, ObjectBuilder<LanguageAnalyz
      */
     @Override
     public Analyzer.Kind _analyzerKind() {
-        return Analyzer.Kind.Language;
+        return Analyzer.Kind.valueOf(this.language.name());
```

Comment:
I think we don't need to change analyzer kind here, we need to change serialization (type -> language) and deserialization (`Analyzer` class):

```
        for (final Language language: Language.values()) {
            op.add(Builder::language, LanguageAnalyzer._DESERIALIZER, language.jsonValue());
        }
```